### DISCUSSION TITLE
Continuation for PDE models

### DISCUSSION THREAD

**Discussion by gotencoder** [adtzlr/contique#16] [General]
Hi there!

Does this package support continuation for PDE based models and are there any examples of this?

Many thanks

**Comment by adtzlr** [comment]
Something like [this](https://felupe.readthedocs.io/en/latest/examples/inflation.html)?

**Reply by gotencoder** [reply]
Hi,

I was thinking more on the lines of a user-defined PDE system (such as Burger's equation). 1) What steps do I need to follow to use contique to perform automated bifurcation analysis? 2) If I have an external solver for a model (either in python or fortran) can that be imported and used with contique?

Many thanks

**Reply by adtzlr** [reply]
1) No, I haven't considered a bifurcation analysis yet.

2) Yes, like in the example. However, you need to have a function for the jacobian w.r.t. the continuation parameter.

**Comment by adtzlr** [comment]
The included test of the [Bratu](https://github.com/adtzlr/contique/blob/main/tests/test_bratu.py) problem could also be of interest.
